### ISSUE TITLE
Browser-native password field "show password" button not keyboard accessible, WCAG, UAAG, or both?

### ISSUE THREAD

**Issue by jacob-willden** [w3c/wcag#4744]
If you have a page with a native password input (`<input type="password">`), Firefox now adds a "show password" button to it, but this button cannot be navigated to via tabbing.

[A bug has been filed for this](https://bugzilla.mozilla.org/show_bug.cgi?id=1999052) on Firefox's bug tracker, but until it is resolved, I'm curious over whether this violates only WCAG, only UAAG, or both. This goes off the general idea of [whether the responsibility of user agent accessibility failures falls on Web authors or not](https://github.com/w3c/wcag/issues/866) (which [appears to not be universally agreed upon itself](https://github.com/w3c/wcag/issues/1009)), but this is more specific.

Adrian Roselli has argued that the [user agent accessibility failures for default field validation falls under WCAG](https://adrianroselli.com/2019/02/avoid-default-field-validation.html), but in the case of default field validation, that issue can be [resolved with some workarounds](https://www.tpgi.com/required-attribute-requirements/). In this case, I do not know of any way to override this default behavior, besides resorting to making a custom password field using ARIA (which would likely result in worse accessibility).

This distinction becomes all the more important with directives like the European Accessibility Act and the new United States rule on Title II of the Americans with Disabilities Act.

Thanks in advance for your time and help!

**Comment by patrickhlauke** [comment]
sounds like a UAAG failure to me ... if only browsers cared about those (ps no need to explain why something is important in light of EAA etc ... we're all well aware of legislation around these parts)

**Comment by jacob-willden** [comment]
Makes sense! Thanks for the clarification!
